src/ast/mod.rs

```diff
@@ -1040,9 +1040,7 @@ pub enum Statement {
         // Specifies the table to merge
         table: TableFactor,
         // Specifies the table or subquery to join with the target table
-        source: Box<SetExpr>,
-        // Specifies alias to the table that is joined with target table
-        alias: Option<TableAlias>,
+        source: TableFactor,
```

Comment:
@mobuchowski What do you think about this change?

Reply:
Looks good to me 👍